providers/implementations/kdfs/pbkdf1.c

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright 1999-2021 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
1999-2021 really?

Reply:
This is the date on the p5_crypt.c file where some of this code is extracted from.